### PR TITLE
Flag conversion modal

### PR DESCRIPTION
**Pull request by mattkrick**
TEST
- [ ] as an admin, call call a mutation like this:

```graphql
mutation {
	flagConversionModal(orgId:"WbqE2L3Cc", active:true) {
    error {
      message
    }
  }
}
```

- [ ] as a member of a team of the flagged org, start a new retro & advance to the discuss phase. see the CC modal
- [ ] cannot escape the CC modal by clicking anything except pay now or pay later
- [ ] other team members in the meeting see a modal saying waiting on the facilitator
- [ ] clicking pay later shows an UpgradeLater modal that cannot be escaped unless you click back to business
- [ ] entering a CC & clicking pay now shows an UpgradeSuccess modal that cannot be escaped unless you click back to business
- [ ] clicking back to business clears the waiting modal for other team members
- [ ] clicking back to business triggers a PayLater mutation that in turn triggers a segment event

### CODE REVIEW

**Comment by jordanh** [comment]
# Test cycle 1

TEST
- [ ] as an admin, call call a mutation like this:

```graphql
mutation {
	flagConversionModal(orgId:"WbqE2L3Cc", active:true) {
    error {
      message
    }
  }
}
```

- [ ] as a member of a team of the flagged org, start a new retro & advance to the discuss phase. see the CC modal
- [ ] cannot escape the CC modal by clicking anything except pay now or pay later
- [ ] other team members in the meeting see a modal saying waiting on the facilitator
- [ ] clicking pay later shows an UpgradeLater modal that cannot be escaped unless you click back to business
- [ ] entering a CC & clicking pay now shows an UpgradeSuccess modal that cannot be escaped unless you click back to business
- [ ] clicking back to business clears the waiting modal for other team members
- [ ] clicking back to business triggers a PayLater mutation that in turn triggers a segment event

File: packages/server/graphql/mutations/payLater.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      .update((row) => ({
+        payLaterClickCount: row('payLaterClickCount').default(0).add(1)
+      }))
+    await r.table('NewMeeting').get(meetingId)
+      .update({
+        showConversionModal: false
+      })
+
+    const organization = await dataLoader.get('organizations').load(orgId)
+    sendSegmentEvent('Conversion Modal Pay Later Clicked', viewerId, {
```
Comment (jordanh) [inline packages/server/graphql/mutations/payLater.ts]: This is perfect! ✨

**Comment by jordanh** [comment]
@mattkrick one little 🐛 on the success modal, merge when fixed!

**Comment by mattkrick** [comment]
@jordanh ready to retest

**Pull request merged**